packages/core/src/shared/settings.ts

```diff
@@ -727,6 +727,7 @@ const devSettings = {
     codecatalystService: Record(String, String),
     codewhispererService: Record(String, String),
     ssoCacheDirectory: String,
+    enableIamPolicyChecksFeature: Boolean,
```

Comment:
You may want to enable this is an "experiment" instead of a dev-only feature. See https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#preview-releases-and-experiments

Reply:
I will enable it as an experiment. Thanks for the tip!